Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -27,6 +27,26 @@ Before the testing library can be used, it must be added as a dependency of your
 Swift package or Xcode project. For more information on how to add it, see the
 [Getting Started](doc:TemporaryGettingStarted) guide.
 
+### Importing the testing library
+
+To import the testing library, add the following to the Swift source file that
```

Comment:
```suggestion
-To import the testing library, add the following to the Swift source file that
-will contain the test:
+XCTest and the testing library are available from different modules. Instead of
+importing the XCTest module, import the Testing module:
```

Reply:
Great, I've changed to to that.